src/Cryptol/REPL/Command.hs

```diff
@@ -2068,14 +2066,23 @@ checkBlock = isolated . go
             , srResult = emptyCommandResult { crSuccess = False }
             }]
         Just (Command cmd) -> do
-          (logtxt, result) <- captureLog (cmd 0 Nothing)
-          results <- checkBlock block
-          let subresult = SubcommandResult
-                { srInput = line
-                , srLog = logtxt
-                , srResult = result
-                }
-          pure (subresult : results)
+          (logtxt, eresult) <- captureLog (Cryptol.REPL.Monad.try (cmd 0 Nothing))
+          case eresult of
+            Left e -> do
+              let result = SubcommandResult
+                    { srInput = line
+                    , srLog = logtxt ++ show (pp e) ++ "\n"
```

Comment:
Why use a newline here in the `Left` case (but not in the `Right` case below)?

Reply:
The newline is terminating the shown error `show (pp e) ++ "\n"` but nothing is being appended to the `Right` case